src/lib/c_fortran_dgssv.c

```diff
@@ -112,25 +103,23 @@ c_fortran_dgssv_(int *iopt, int *n, int *nnz, int *nrhs,
 	relax = sp_ienv(2);
 
 	dgstrf(&options, &AC, relax, panel_size, etree,
-                NULL, 0, perm_c, perm_r, L, U, &Glu, &stat, info);
+                NULL, 0, perm_c, perm_r, L, U, &stat, info);
```

Comment:
This change needs to be explained - the commited code is compatable eith modern superlu and should not be changed if this is to allow you to use earlier versions of superlu